src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -1085,9 +1222,299 @@ .maxstack 8
 	}
 } // end of class C
 ");
+            Assert.Equal(1, accessorBindingData.NumberOfPerformedAccessorBinding);
```

Comment:
>Assert.Equal(1, accessorBindingData.NumberOfPerformedAccessorBinding); [](http://example.com/codeflow?start=12&length=70)

We should see if we can get this back to zero.

Reply:
#Pending

Reply:
If the property is declared before the constructor (ie, compiled first), then we don't do extra binding.

The extra binding here results from the fact that we need to know if this is a semi-auto property for assigning in the constructor.

Reply:
> We can consider compiling all accessors first to avoid the extra binding. But I'm not sure if it's worth doing it. Let me know what do you think.

Responded on a different thread

Reply:
Fixed